osmaxx/utils/polyfile_helpers.py

```diff
@@ -67,3 +67,23 @@ def parse_poly(lines):
             in_ring = True
 
     return MultiPolygon(*(Polygon(*polycoords) for polycoords in coords), srid=4326)
+
+
+def get_polyfile_names_to_file_mapping():
+    """
+    generates names of all polyfiles inside the POLYFILE_LOCATION path.
+
+    Returns: a mapping of polyfile names to polyfile path
+    """
+    from osmaxx.excerptexport._settings import POLYFILE_LOCATION
+    polyfile_name_to_file_path_mapping = {}
+    for root, dirs, files in os.walk(POLYFILE_LOCATION):
+        subfolder_name = root.replace(POLYFILE_LOCATION, '').replace('/', '')
```

Comment:
not very likely, since `POLYFILE_LOCATION` should be an absolute path, but I like the explicitness of your expression :-)